test/data/test_lightning_datamodule.py

```diff
@@ -329,9 +329,7 @@ def test_lightning_hetero_link_data():
     )
     assert isinstance(datamodule.neighbor_sampler, NeighborSampler)
     for batch in datamodule.train_dataloader():
-        assert 'edge_label' in batch['author', 'paper']
```

Comment:
Why remove this check? isn't the default `binary` which adds `edge_label`.

Reply:
We no longer add `edge_label` in case `neg_sampling = None` and no `edge_label` was passed.